.github/workflows/tests.yml

```diff
@@ -56,22 +56,16 @@ jobs:
         with:
           cache-read-only: ${{ github.ref != 'refs/heads/develop' }}
 
-      - name: ⚙️ Run unit tests for debug variant
-        run: ./gradlew testDebugUnitTest $CI_GRADLE_ARG_PROPERTIES
-
-      - name: 📸 Run screenshot tests
-        run: ./gradlew verifyPaparazziDebug $CI_GRADLE_ARG_PROPERTIES
-
-      - name: 📈Generate kover report and verify coverage
-        run: ./gradlew :app:koverXmlReportGplayDebug :app:koverHtmlReportGplayDebug :app:koverVerifyAll $CI_GRADLE_ARG_PROPERTIES
+      - name: ⚙️ Check coverage for debug variant (includes unit & screenshot tests)
+        run: ./gradlew :tests:uitests:verifyPaparazziDebug :app:koverXmlReportGplayDebug :app:koverHtmlReportGplayDebug :app:koverVerifyAll $CI_GRADLE_ARG_PROPERTIES
```

Comment:
I don't see testDebugUnitTest inside?

Reply:
It's not needed, the kover tasks already run them. You can check the logs of the GH actions to confirm.